crates/core/src/otel.rs

```diff
@@ -137,6 +137,7 @@ impl OtelConfig {
 }
 
 #[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
```

Comment:
Should we change the `OtelSignal` enum here as well? Or really just all things that we serialize?

Reply:
I would stick to just to the things that we serialize, `OtelSignal` is intended for the library's internal use only, so I don't see the need to rename